src/Engine/MapController.java

```diff
@@ -166,7 +168,10 @@ private void handleMapInput(InputHandler.InputAction input)
       case SEEK: // Move the cursor to the next unit that is ready to move.
         boolean found = false;
         int tries = 0;
-        int maxTries = myGame.activeCO.units.size();
+        int numUnits = myGame.activeCO.units.size();
+        ArrayList<XYCoord> freeIndustries = Utils.findFreeIndustries(myGame.activeCO, myGame.gameMap);
```

Comment:
As for naming, since you mentioned it. Maybe `UsableProperties` instead of `FreeIndustries`? It's a little awkward, but is at least more descriptive, I think.